### ISSUE TITLE
Remoção da função subtração

### ISSUE THREAD

**Issue by danielsalles** [Webschool-io/matematica-para-programadores#10]
Eu li seu readme e ele abriu minha mente.
Acho que a função subtração não deveria existir, já que a função não passa de uma soma. Na minha opinião ela foi criada para o melhor entendimento, mas na verdade só causa complicação. Se eu quero fazer `1 - 1`, isso pode ser convertido para:
`soma(1, -1)` ou seja, `soma = 1 + (-1)`. Faz total sentido, como o @suissa já deixou explicito na suas explicações. Acho que não é necessário ter uma função para isso, seria como escrever a função soma duas vezes.
Claro, que as linguagens possuem o operador de `subtração -`, mas depois que passei a entender, nunca mais irei usar esse operador, pois não faz sentido lógico para mim.

Acho que fui um pouco exagerado, mas foda-se uhahuahua. Estou aqui só para debater a remoção dessa função.

**Comment by suissa** [comment]
Concordo plenamente, até já expliquei como fazer isso aqui https://github.com/Webschool-io/matematica-para-programadores/blob/master/basico/apenas-soma.md

**Comment by suissa** [comment]
Essa já foi feita neh @danielsalles ?
